app/scripts/components/common/mapbox/map-options/use-basemap.ts

```diff
@@ -0,0 +1,112 @@
+import mapboxgl, { Layer } from 'mapbox-gl';
+import { useCallback, useEffect, useMemo, useState } from 'react';
+import {
+  BasemapId,
+  BASEMAP_STYLES,
+  GROUPS_BY_OPTION,
+  Option
+} from './basemaps';
+
+export function useBasemap(mapInstance: mapboxgl.Map | null) {
+  const [styleLoaded, setStyleLoaded] = useState(true);
```

Comment:
Shouldn't this start as `false`? I think it may be participating in the error mentioned on line 99.